App/Sources/Presentation/TeacherMain/Source/Scene/SignupListPopupView.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftUI
+
+struct SignupListPopupView: View {
+    @Binding var showPopup: Bool
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Spacer()
+                Button(action: {
+                    showPopup = false
+                }) {
+                    Image(systemName: "xmark")
+                        .foregroundColor(.black)
+                        .padding(.top, 18)
+                        .padding(.trailing, 18)
+                }
+            }
+            .frame(height: 50)
+            
+            ScrollView(.vertical, showsIndicators: false) {
+                VStack(spacing: 10) {
+                    ForEach(1..<4) { _ in
+                        VStack {
+                            HStack(spacing: 10) {
+                                Spacer()
+                                    
+                                Text("") // TeacherMain페이지 model 완성 후 그에 맞는 데이터 삽입 필요
+                                    .skFont(.pm16)
+                                
+                                Text("-")
+                                
+                                Text("") // TeacherMain페이지 model 완성 후 그에 맞는 데이터 삽입 필요
```

Comment:
```suggestion
                                Text("") // TeacherMain페이지 model 완성 후 그에 맞는 데이터 삽입 필요
```
주석이 있는데 무슨 뜻인지 이해를 못하겠어요...

Reply:
TeacherMainView에 container가 만들어지고 나서 저 Text의 "" 자리에 걸맞는 걸 삽입한다는 뜻이었습니다!

Reply:
저러한 의도 였다면 주석을 어떤식으로 다는게 좋을까요?? 의견 여쭤보고 싶습니다~

Reply:
PopupView는 container가 필요 없는 CustomView 아닌가요? TeacherMain의 Model이 만들어져야 하는 이유가 뭔지 궁금합니다. 삽입 한다는 말이 조금 애매하기도 해서 CustomView면 ""보다는 상수나 변수를 선언해서 넣어두는 편이 더 좋을 것 같아요

Reply:
저는 그냥 CustomView 각각마다 container가 들어가야되는줄 알았습니다~ 그럼 그렇게 수정해서 다시 올리겠습니다~